OloEditor/src/Panels/SceneHierarchyPanel.cpp

```
@@ -41,6 +41,17 @@
 #include <unordered_map>
 #include <algorithm>
 
+namespace
+{
+    // Interprets ImGui drag-drop payload bytes as a UTF-8 path.
+    [[nodiscard]] std::filesystem::path PathFromUtf8Payload(const ImGuiPayload& payload)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)